flatpak/io.github.sezanzeb.KeyMapper.yml

```diff
@@ -0,0 +1,23 @@
+app-id: io.github.sezanzeb.KeyMapper
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: /app/bin/key-mapper.sh
+
+finish-args:
```

Comment:
`  - --device=all` is needed to see anything in the gui, but that won't make all devices visible unfortunately

Reply:
But it still fails at injecting stuff

```
Process Process-1:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/app/usr/lib/python3.8/site-packages/keymapper/dev/injector.py", line 439, in _start_injecting
    path, uinput.capabilities(verbose=True)
  File "/app/lib/python3.8/site-packages/evdev/uinput.py", line 224, in capabilities
    raise UInputError('input device not opened - cannot read capabilities')
evdev.uinput.UInputError: input device not opened - cannot read capabilities
```

which is most likely caused by missing permissions on `/dev/uinput`

Reply:
> No access to any device nodes

I think this is the problem https://docs.flatpak.org/en/latest/sandbox-permissions.html

Reply:
@sezanzeb Have you tried this option?
> --device=all

`--device=DEVICE`
Expose a device to the application. This updates the [Context] group in the metadata. DEVICE must be one of: dri, kvm, shm, all. This option can be used multiple times.

Reply:
According to https://docs.flatpak.org/en/latest/flatpak-command-reference.html this should work
> All device nodes in /dev, but not /dev/shm (which is separately specified). Available since 0.6.6.

Reply:
Sorry, misread your message. If `device=all` doesn't work - could you open bug in Flatpak describing original intent of application.

Reply:
https://github.com/flatpak/flatpak/issues/4137